recipes/dammit/meta.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: dammit
-  version: '0.3'
+  version: '0.3.2'
 
 source:
-  url: https://github.com/camillescott/dammit/archive/v0.3.tar.gz 
-  fn: dammit-v0.3.tar.gz
+  url: https://github.com/camillescott/dammit/archive/v0.3.2.tar.gz 
+  fn: dammit-v0.3.2.tar.gz
   md5: e7d3c00ab88c81a3a27088b14ab5472f
```

Comment:
Hi @mictadlo, the file has changed so the checksum needs to be updated. You might as well replace the md5 with a sha256. You can replace with the following line:

    sha256: 382cdc04d47ca230c0843472b17a095ec102707067db6b9d093a8289d993f959